src/xss-filters.js

```diff
@@ -351,23 +357,22 @@ exports._getPrivFilters = function () {
         // * http://www.w3.org/TR/CSS21/grammar.html 
         // * http://www.w3.org/TR/css-syntax-3/
         // 
-        // NOTE: delimitar in CSS - \ _ : ; ( ) " ' / , % # ! * @ . { }
+        // NOTE: delimiter in CSS -  \  _  :  ;  (  )  "  '  /  ,  %  #  !  *  @  .  {  }
+        //                        2d 5c 5f 3a 3b 28 29 22 27 2f 2c 25 23 21 2a 40 2e 7b 7d
 
-        // CSS_UNQUOTED_CHARS = /[^%#+\-\w\.]/g,
         yceu: function(s) {
-            return css(s, CSS_UNQUOTED_CHARS);
+            s = htmlDecode(s);
+            return CSS_VALID_VALUE.test(s) ? s : ";-x:'" + cssBlacklist(s.replace(CSS_SINGLE_QUOTED_CHARS, cssEncode)) + "';-v:";
```

Comment:
you can say so. or in other words, whatever passed the restrictive CSS_VALID_VALUE is safe to be left untouched. for those failed ones, they're put inside a quoted CSS value, and receive the standard CSS value treatment. the latter arrangement does not create any new attack surface (or new regexp).